rwengine/src/engine/SaveGame.cpp

```diff
@@ -1226,6 +1226,7 @@ bool SaveGame::loadGame(GameState& state, const std::string& file) {
         state.script->startThread(scripts[s].programCounter);
         SCMThread& thread = threads.back();
         // thread.baseAddress // ??
+        // husho: baseAddress does not exist in III and VC
```

Comment:
nit: comment should be on same line or above.
Also "// Only in SA" would be a better comment (and LCS / VCS if that affects it).

Reply:
I didnt write only in SA, because I dont know how its implemented in SA